src/routes/prod.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import { docsPerBuildIdHandler } from './builds';
+import { projectBranchDocsHandler, timestampDocsHandler } from './projects';
+
+const router = express.Router();
+
+router.get('/projects/:snootyProject/:branch/documents', projectBranchDocsHandler);
```

Comment:
(No change needed, asking for my own education) I like the use of request handlers! Since the routes and logic are the same, I'm curious why reuse handlers instead of reusing the routers?

Reply:
more of the former suggestion. however, the differentiation logic is present in both handlers so i am opening to re-use routers, but unclear of this method. ie.

in `projects.ts` or `builds.ts`
```
export const route = router.get('/:snootyProject/:branch/documents', handlerFnHere)
```

how would we set the prod prefixed path with this route, was unclear to me

Reply:
Since you have the router for `/prod`, inside of that router, you should be able to do something like `router.use('/projects', projectsRouter)`, similar to what's currently done in the main app file. The final route should end up being the same

Reply:
very clean idea! done. removed route file for `prod.ts` and re-using the router